ui/tutorial-renderer/src/main/res/layout/optin_onboarding_explore_slide.xml

```diff
@@ -0,0 +1,88 @@
+    </data>
+
+    <androidx.constraintlayout.widget.ConstraintLayout
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"
+        android:orientation="vertical">
+
+        <include
+            layout="@layout/optin_onboarding_navigation_view"
+            app:callback="@{callback}" />
+
+        <androidx.appcompat.widget.AppCompatTextView
+            android:id="@+id/onboarding_explore_text"
+            android:layout_width="0dp"
+            android:layout_height="wrap_content"
+            android:layout_marginTop="32dp"
+            android:gravity="bottom|center_horizontal"
+            android:text="@string/onboarding_watch"
+            android:textAppearance="@style/TextAppearance.GodTools.Large"
+            android:textColor="@color/design_default_color_primary_dark"
+            tools:textColor="@color/gt_blue_dark"
+            app:layout_constraintBottom_toTopOf="@+id/onboarding_video_slide_text"
+            app:layout_constraintEnd_toEndOf="parent"
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintTop_toBottomOf="@+id/onboarding_previous_button" />
+
+        <androidx.appcompat.widget.AppCompatTextView
+            android:id="@+id/onboarding_video_slide_text"
+            android:layout_width="0dp"
+            android:layout_height="wrap_content"
+            android:layout_marginStart="32dp"
+            android:layout_marginLeft="32dp"
+            android:layout_marginTop="8dp"
+            android:layout_marginEnd="32dp"
+            android:layout_marginRight="32dp"
+            android:gravity="center"
+            android:text="@string/onboarding_slide_1_description"
+            android:textAppearance="@style/TextAppearance.GodTools.Medium"
+            app:layout_constraintBottom_toTopOf="@+id/onboarding_youtube_container"
+            app:layout_constraintEnd_toEndOf="parent"
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintTop_toBottomOf="@+id/onboarding_explore_text" />
+        <LinearLayout
+            android:id="@+id/onboarding_youtube_container"
+            android:layout_width="0dp"
+            android:layout_height="0dp"
+            android:layout_marginStart="32dp"
+            android:layout_marginLeft="32dp"
+            android:layout_marginTop="16dp"
+            android:layout_marginEnd="32dp"
+            android:layout_marginRight="32dp"
+            android:layout_marginBottom="32dp"
+            android:gravity="top|center_horizontal"
+            android:orientation="vertical"
+            app:layout_constraintBottom_toTopOf="@+id/onboarding_next_button"
+            app:layout_constraintEnd_toEndOf="parent"
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintTop_toBottomOf="@+id/onboarding_video_slide_text">
+
+            <!-- The YouTube player is in a linear layout because you can't set gravity on it. -->
```

Comment:
gravity is probably the wrong attribute for whatever you were trying to do. There shouldn't be need to nest this is another layout